grpc-testtool/proto/merkle/merkle.proto

```diff
@@ -0,0 +1,102 @@
+syntax = "proto3";
+
+package merkle;
+
+import "google/protobuf/empty.proto";
+
+service Merkle {
+    rpc NewProposal(NewProposalRequest) returns (NewProposalResponse);
+    rpc ProposalCommit(ProposalCommitRequest) returns (ProposalCommitResponse);
+  
+    rpc NewView(NewViewRequest) returns (NewViewResponse);
+    // The methods below may be called with a view ID that corresponds to either a (committable) proposal
+    // or (non-committable) historical view.
+    rpc ViewHas(ViewHasRequest) returns (ViewHasResponse);
+    rpc ViewGet(ViewGetRequest) returns (ViewGetResponse);
+    rpc ViewNewIteratorWithStartAndPrefix(ViewNewIteratorWithStartAndPrefixRequest) returns (ViewNewIteratorWithStartAndPrefixResponse);
+    rpc ViewRelease(ViewReleaseRequest) returns (google.protobuf.Empty);
+}
+
+message NewProposalRequest {
+  // If not given, the parent view is the current database revision.
+  optional uint32 parent_view_id = 1;
+  repeated PutRequest puts = 2;
+  repeated bytes deletes = 3;
+}
+
+message NewProposalResponse {
+  uint32 proposal_id = 1;
+}
+
+message ProposalCommitRequest {
+  uint32 proposal_id = 1;
+}
+
+message ProposalCommitResponse {
+  CommitError err = 1;
+}
+
+message NewViewRequest {
+  bytes root_id = 1;
+}
+
+message NewViewResponse {
+  // If not given, the parent view is the current database revision.
+  optional uint32 view_id = 1;
```

Comment:
I still want a `view_id` even if it's the current database revision at the time of this call.

This means I can continue to use the view even if I commit more stuff

Reply:
This was a copy-paste error. Now it always returns a `view_id`.